src/overlays/actors/ovl_Dm_Hina/z_dm_hina.c

```diff
@@ -15,12 +16,19 @@ void DmHina_Destroy(Actor* thisx, PlayState* play);
 void DmHina_Update(Actor* thisx, PlayState* play);
 void DmHina_Draw(Actor* thisx, PlayState* play);
 
-void func_80A1F470(DmHina* this, PlayState* play);
-void func_80A1F56C(DmHina* this, PlayState* play);
-void func_80A1F5AC(DmHina* this, PlayState* play);
-void func_80A1F63C(DmHina* this, PlayState* play);
+void DmHina_WaitForPlayer(DmHina* this, PlayState* play);
+void DmHina_AwaitMessageBoxClosing(DmHina* this, PlayState* play);
+void DmHina_SetupSubCamera(DmHina* this, PlayState* play);
+void DmHina_MoveSubCamera(DmHina* this, PlayState* play);
+
+#define PARAMS_REMAINS_ODOLWA 0
+#define PARAMS_REMAINS_GOHT 1
+#define PARAMS_REMAINS_GYORG 2
+#define PARAMS_REMAINS_TWINMOLD 3
```

Comment:
Seems better as an enum.

Reply:
I had these as defines since they were related to params, does that make a difference?

Reply:
I'd still say enum. It looks like the `Type` of remains. It just uses params directly because it doesn't use params for anything else, so it doesn't need to piece it out.

Reply:
Makes sense 👍